magpie/utils.py

```diff
@@ -5,6 +5,7 @@
 except ImportError:
```

Comment:
For me in order to review this PR, can you explain better what you did in this commit log? What were these `forgotten functions` and in which sense you have updated them?